### PR TITLE
fix(MediaSettings): adjust transition direction based on language sup…

### PR DESCRIPTION
**Pull request by DorraJaouad**
…port
### ☑️ Resolves

* Fix this

https://github.com/user-attachments/assets/3fa112a9-a7ba-47ee-9358-ec4328e4d84b

🏡 After

https://github.com/user-attachments/assets/05785de3-cdbd-4e7c-b3bd-6476b6219655

### 🏁 Checklist

- [] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [ ] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/MediaSettings/MediaSettingsTabs.vue [hunk 1]
```diff
@@ -29,6 +31,8 @@ const emit = defineEmits<{
 /** Whether the tab panel is open */
 const isOpen = ref(!!props.active)
 
+const isRTLDirection = computed(() => isRTL(getLanguage()))
```
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/components/MediaSettings/MediaSettingsTabs.vue:34]: And we don't have dynamic language change, so you can omit `computed` and just store result in const

[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/MediaSettings/MediaSettingsTabs.vue:34]: IK, but it was not costly to keep it reactive. Yet, reverting not to confuse others that it's dynamic

**Review by Antreesy — APPROVED** [review]
Tested

**Pull request merged**